client/src/pages/LoginPage/LoginPage.js

```diff
@@ -0,0 +1,47 @@
+import React, { useEffect } from "react";
+import CredentialsInputBox from "../../components/CredentialsInputBox/CredentialsInputBox.js";
+import {
+  HeaderText,
+  SubtitleText,
+} from "../../components/ThemeText/ThemeText.js";
+import { CredentialsButton } from "../../components/CredentialsButton/CredentialsButton.js";
+import { DescriptiveTextButton } from "../../components/DescriptiveTextButton/DescriptiveTextButton.js";
+import { useNavigate } from "react-router-dom";
+
+import "./LoginPage.css";
+
+const LoginPage = () => {
```

Comment:
You might be doing this in your next PR (with linking this page to the APIs made) but I would also include the setup where for any text input on either the email/password fields, they should be linked with useState.

If you do include this, then when making your API calls, it'll be much easier to link stuff together!

Reply:
Side note: If you decide to include this portion here, you'll need to use `useState` along with onChange.

i.e. 
I've included an example to how this will look like (syntax might be slightly off)
```
import {useState} from "react";

const [text, setText] = useState("");

const onChangeHandler = (e) => {
    setText(e.target.value);
}
<input onChange={(e) => onChangeHandler(e)} value={text}/>
```